src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,91 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  movies: Movie[];
+  onAdd: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ onAdd, movies }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [searchedQuery, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [isSubmitted, setSubmitted] = useState(false);
+  const isValidData = (data: MovieData | ResponseError): data is MovieData => {
+    return (data as MovieData).Title !== undefined;
+  };
+
+  useEffect(() => {
+    if (searchedQuery) {
+      setIsError(false);
+    }
+  }, [searchedQuery]);
```

Comment:
you can make a handler for `input` instead of `useEffect` and after every change query use `setIsError(false)`

```
const handleQueryInput = (event) => {
      setQuery(event.target.value);
      setIsError(false);
    };
```

Reply:
make it handler in the change function
It's not ok do it in `useEffect`